actions/transactions/routes.go

```diff
@@ -12,12 +12,12 @@ type Action struct {
 	actions.Action
 }
 
-// NewTransactionsHandler creates the specific package routes in restfull style
-func NewTransactionsHandler(appConfig *config.AppConfig, services *config.AppServices) *routes.Handler {
+// NewHandler creates the specific package routes in restfull style
+func NewHandler(appConfig *config.AppConfig, services *config.AppServices) *routes.Handler {
 	action := &Action{actions.Action{AppConfig: appConfig, Services: services}}
 
 	handler := routes.Handler{
```

Comment:
the use of `routes.Handler` is different, but I guess it makes no difference so lets leave it that way

Reply:
I only changed the name - if you have any suggestions feel free - I can update it

Reply:
👀

Reply:
nah, there's no need